scripts/bench.py

```
@@ -0,0 +1,103 @@
+        messages.append({"role": "user", "content": user_prompt})
+
+        # Create 10 concurrent requests
+        tasks = [timed_chat(client, list(messages)) for _ in range(30)]
+
+        # Wait for them all to finish
+        results = await asyncio.gather(*tasks)
+
+        elapsed_times = []
+        t_s = []
+        # Show the responses
+        for i, (completion, elapsed) in enumerate(results, 1):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)